x-pack/plugins/infra/public/pages/metrics/inventory_view/components/show_kubernetes_tour.tsx

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Suggestion: How about naming the file as `kubernetes_tour` and the component `KubernenetesTour`?

The name with a verb gives me the impression that this is a function. Wdyt?